exercises/ex3/README.md

```diff
@@ -118,57 +118,66 @@ After completing these steps, you will have configured credentials for connectin
    - For **Password**, use the personal access token, which you've created in GitHub in exercise 3.3.
    ![Credentials GitHub](./images/CICD_credentials_github.png)
 
-6. To create credentials for deploying to the SAP Cloud Platform Cloud Foundry environment, go to the **Credentials** tab and choose **+** *(Create Credentials)*.
+5. To create credentials for deploying to the SAP Cloud Foundry environment, go to the **Credentials** tab and choose **+** *(Create Credentials)*.
 ![Credentials](./images/CICD_credentials.png)
 
-7. For **Name**, enter a freely chosen name for your credentials, which is unique in your SAP Cloud Platform subaccount. In this example, the name of the credentials is *cfdeploy*.
+6. For **Name**, enter a freely chosen name for your credentials, which is unique in your SAP BTP subaccount. In this example, the name of the credentials is *cfdeploy*.
 
-8. As **Type**, select **Basic Authentication**.
+7. As **Type**, select **Basic Authentication**.
 
-9. For **Username**, enter your username for the SAP Cloud Platform cockpit.
+8. For **Username**, enter your username of your SAP BTP cockpit.
```

Comment:
username *for* the SAP SAP BTP cockpit

username of *your* SAP BTP cockpit is misleading from my perspective